### DISCUSSION TITLE
Benefits Pending

### DISCUSSION THREAD

**Discussion by KotraHaridutt** [community/community#145978] [GitHub Education]
### Hi everyone,

My request got approved on November 29th.. And did not receive the mail.. Waiting for the benefits to access and to explore and all the features.. Thank you

**Comment by GabJuarez** [comment]
Hi!

It usually takes up to a week for the benefits of the Student Developer Pack to be activated after approval. If it has been more than 7 days and you still haven’t received them, I recommend contacting GitHub Education Support directly via this link: 

[Support](https://support.github.com/contact/education)

Hope this helps!

Let me know if you’d like to tweak it further. (If I helped you please mark me as an answer😸)

**Comment by davevad93** [comment]
Hi @KotraHaridutt ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @KotraHaridutt was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
